src/modules/sessions-management-pane/management-new-session/management-new-session.tsx

```diff
@@ -30,13 +30,15 @@ interface Props {
   serviceGroupId: string;
   hasGlobalSession: boolean;
   setFieldError: (field: string, errorMsg: string) => void;
+  setErrors: (fields: { [field: string]: string }) => void;
```

Comment:
Check the documentation of formik, maybe you cam use this functions from useFormik or something else